packages/dingus/README.md

```diff
@@ -0,0 +1,16 @@
+# TemplateMark Dingus
+
+An online tool which lets you edit the markdown and see it rendered as HTML, or as a document object model. This is [deployed on Netlify at this link](https://templatemark-dingus.netlify.app/).
+
+Further documentation can be found at [this link](https://docs.accordproject.org/docs/markup-preliminaries.html#dingus).
+
+## Installation
```

Comment:
I would remove `usage` which may not make sense.. (it's just a demo / testing thing).

In `Installation` (maybe better named `run locally`) :

```
npm run dingus
open demo/index.html
```